packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -478,14 +479,14 @@ const EAGivingPortalPage = ({classes}: {classes: ClassesType}) => {
                 Donate, <span className={classes.primaryText}>Discuss</span>, Vote
               </div>
               <div className={classes.h4}>
-                Recent posts tagged Donation Election 2023
+                Recent posts tagged Effective giving
```

Comment:
I would prefer to have quotes around "Effective giving" here, it's bit confusing that only the word "Effective" is capitalized (but nbd either way)